monitoring/README.md

```diff
@@ -107,7 +111,7 @@ wds1-ks-transport-controller   64m
 b) ITS space:
 
 ```bash
-./configure-monitoring-its.sh 
+./configure-monitoring-its.sh --host-cluster-context kind-kubeflex
 ```
 
```

Comment:
BTW, the set of Grafana dashboards in the picture at new line 174 is outdated; there I see 3, but now there are 4.

```console
mspreitz@mjs13 monitoring % ls grafana/*.json
grafana/APIServerMonitoring.json			grafana/KubeStellarControllersMonitoring-Average.json
grafana/APIServerPriorityFairness.json			grafana/KubeStellarControllersMonitoring.json
```

Reply:
Yes, the newly added Grafana dashboard will be useful for debugging for me it some situations but I don't want to expose it to the end-user. Hence, intentionally, I did not update the documentation to reference this new Grafana dashboard.

Reply:
But you have two references, one is updated and one is not. One refers the reader to everything in the directory, the other shows a snapshot of the result of loading those.

Reply:
Ok, I see! To avoid any further confusion, I removed the file `KubeStellarControllersMonitoring-Average.json`